server/liveblog/blogs/blog.py

```diff
@@ -90,6 +91,11 @@ def fix_item_markup_if_needed(self, doc):
         if doc.get("type") == "text":
             original_text = doc["item"].get("text")
             doc["item"]["text"] = self.check_html_markup(original_text)
+        elif doc.get("type") == "poll":
+            doc["item"]["poll_body"]["active_until"] = doc["item"]["poll_body"][
+                "active_until"
+            ].isoformat()
```

Comment:
As we spoke earlier today, please add some "safe guard" for potential `None` values.